specification/translation/Azure.AI.DocumentTranslation/main.tsp

```diff
@@ -0,0 +1,233 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@useAuth(ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">)
+@service({
+  title: "Azure.AI.DocumentTranslation",
+})
+@versioned(DocumentTranslation.Versions)
+namespace DocumentTranslation;
+
+/** The Azure.AI.DocumentTranslation versions. */
+enum Versions {
+  /** The 2023-11-01-preview version of the Azure.AI.DocumentTranslation. */
+  @useDependency(Azure.Core.Versions.v1_0_Preview_1)
+  `2023-11-01-preview`,
+}
+
+alias CommonParameters = {
+  /** Mandatory API version parameter */
+  @query("api-version")
+  apiVersion: string = "2023-11-01-preview";
+
+  /** The value is the region of the translator resource. This value is optional if when custom endpoint is used.
+   * The value can be found in Azure portal. Translator resource => Resource Management => Keys and Endpoint => Location/Region  */
+  @header("Ocp-Apim-Subscription-Region")
+  subscriptionRegion?: string;
+
+  /** A client-generated GUID to uniquely identify the request. */
+  @header("X-ClientTraceId")
```

Comment:
Does this have precedent for Cognitive/AI services already? We recommend `x-ms-client-request-id`, which our SDKs support natively. Customers don't have to specify it, but can; otherwise, we generate one.